distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -79,3 +79,17 @@ grant codeBase "file:/pax-logging-log4j2" {
     //  permission java.io.FilePermission "/the/new/log/directory/-", "read, write";
     //  permission java.io.FilePermission "/the/new/log/directory", "read, write";
 }
+
+//                       SSH Permissions
+//  Adding required permissions for commands executed via SSH.
+//
+//  Access to files outside of the user or karaf home directories is restricted when using SSH. To allow a command to access a directory
+//  when using SSH, permission entries need to be added in the following field.
+//
+grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.karaf.sshd.core" {
+    //  Add required permissions here.
+
+    //  Example:
```

Comment:
❓ Is it a requirement to add both the general directory permission and the directory wildcard permission in order to ingest from that directory?

If so, it may be nice to make an extra note about that (for lazy people like me that don't want to make _2_ directory permissions 😄).